services/utils.go

```diff
@@ -0,0 +1,29 @@
+package services
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"io"
+	"os"
+	"sha256-sum/errors"
+)
+
+func HashOfFile(path string) string {
+	file, err := os.Open(path)
+	if err != nil {
+		errors.CheckErr(err)
+		return ""
+	}
+
+	defer file.Close()
+
+	hash := sha256.New()
+	_, err = io.Copy(hash, file)
+
+	if err != nil {
+		errors.CheckErr(err)
+		return ""
+	}
+	res := "File: " + file.Name() + ", Checksum: " + hex.EncodeToString(hash.Sum(nil))
```

Comment:
Its better to follow shasum binary standard:

Example:

$ shasum -a 256 jmeter.log

fa938197509566cce2cb6499a08e8583d3bfc7e340cd6427e5de1641aa7ec615 jmeter.log